packages/@ama-sdk/core/schematics/ng-add/index.ts

```diff
@@ -1,29 +1,23 @@
 import { chain, type Rule } from '@angular-devkit/schematics';
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
-import { createSchematicWithMetricsIfInstalled } from '@o3r/schematics';
 import * as path from 'node:path';
 import * as ts from 'typescript';
 import type { NgAddSchematicsSchema } from './schema';
 
+const reportMissingSchematicsDep = (logger: { error: (message: string) => any }) => (raison: any) => {
+  logger.error(`[ERROR]: Adding @ama-sdk/core has failed.
+      If the error is related to missing @o3r dependencies you need to install '@o3r/schematics' to be able to use the @ama-sdk/core ng add. Please run 'ng add @o3r/schematics' .
+      Otherwise, use the error message as guidance.`);
+  throw raison;
+};
```

Comment:
```suggestion
const reportMissingSchematicsDep = (logger: { error: (message: string) => any }) => (reason: any) => {
  logger.error(`[ERROR]: Adding @ama-sdk/core has failed.
      If the error is related to missing @o3r dependencies you need to install '@o3r/schematics' to be able to use the @ama-sdk/core ng add. Please run 'ng add @o3r/schematics' .
      Otherwise, use the error message as guidance.`);
  throw reason;
};
```

Reply:
oops 🍭